app/controllers/bookmarks_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class BookmarksController < CatalogController
+  include Blacklight::Bookmarks
+
+  configure_blacklight do |_config|
```

Comment:
```suggestion
  configure_blacklight do |config|
```

An underscore before a var is, I believe, only used on vars that don't get called. Right?